### PR TITLE
update Test environment

### PR DESCRIPTION
**Pull request by erwanp**
@minouHub this should make tests faster : 
- only one Test stage; i.e. test environment is built only one
- however Fast Tests are ran first, which I think was the intent behind splitting in two test stages

### CODE REVIEW

**Comment by minouHub** [comment]
Hello. A quick reply: is the build failing as soon as the fast tests are failing. I might have missed something a few months ago but the build would run all the commands even if the fast tests failed

**Comment by minouHub** [comment]
See comment here: https://github.com/travis-ci/travis-ci/issues/1066#issuecomment-570898290

**Comment by erwanp** [comment]
Indeed you're right, tests continued even after failure of fast tests : https://app.travis-ci.com/github/radis/radis/jobs/623023907#L2177

**Comment by erwanp** [comment]
It worked correctly and failed after 3min https://app.travis-ci.com/github/radis/radis/jobs/623146582

**Comment by minouHub** [comment]
Ok, I'm convinced. It's a good idea to include fast_tests in the `before_script` section.

**Comment by minouHub** [comment]
@erwanp the Travis file is now updated to test Python 3.8, 3.9, 3.10. Please comment, and thick the following if you agree.

- [ ] We can add `python=3.11` that is supposed to be problematic with `vaex` in #656 . 
- [ ] We can still add some testing on Windows. Don't know if that relevant.

Concerning `cbuildwheel`, there was an error because Radis is "a pure Python project". Using `cbuildwheel` was mentioned with @dcmvdbekerom in https://github.com/radis/radis/issues/596#issuecomment-1665532986. Since that comment `cython` is not used anymore. Thus, I just commented the line and replaced it with a pure Python wheel. I'm completely new to all that jazz so do not hesitate to change it. 
_In short:_ Do we still need `cbuildwheel`?

File: .travis.yml [hunk 1]
```diff
@@ -1,130 +1,121 @@
+# Runs Test scripts; Linting; Deployment on Travis-CI
 language: python
 python:
   # We don't actually use the Travis Python, but this keeps it organized.
   #- "2.7"    # not tested anymore starting from 0.9.19
   - "3.8"
+  - "3.9"
+  - "3.10"
 os: linux
+dist: focal #needed for python 3.10 (xenial is default for Travis-CI, but does not support python 3.10)
```
Comment (erwanp) [inline .travis.yml:10]: that was hard to find ! @minouHub

File: .travis.yml [hunk 2]
```diff
@@ -1,130 +1,121 @@
+# Runs Test scripts; Linting; Deployment on Travis-CI
 language: python
 python:
   # We don't actually use the Travis Python, but this keeps it organized.
   #- "2.7"    # not tested anymore starting from 0.9.19
   - "3.8"
+  - "3.9"
+  - "3.10"
```
Comment (erwanp) [inline .travis.yml:8]: The more Python versions the faster we burn out Credits. I'd keep 1 Vaex-version only; and 1 non-Vaex version (after #656)  @minouHub  

So for the time being , let's test on Python 3.10  and drop 3.8; 3.9 tests ?

In the future we might want to test multiple versions only for merges to Master branch , but I don't know how to add that easily in the Travis matrix.
At some point we'll also switch to GitHub Actions; so let's do this multi-python testing at that time (GSOC project?)

Comment (minouHub) [reply .travis.yml:8]: On the other hand ... if we had the tests ready for python 3.11 and 3.12, we would have caught the problem with vaex earlier. I don't know if the Travis credit are really a problem, we can always ask for a renewal right? On top of that, we now have the "fast" tests stopping the build and saving a lot of credits. I'd prefer to keep several python versions tested. Let's say three at least, distributed the way we prefer.

Comment (minouHub) [reply .travis.yml:8]: On that: 

- [ ] let's drop the tests on Windows for the moment
- [ ] I'll look for a way to implement more tests but only on the master branch

**Comment by minouHub** [comment]
linting failed .... too bad, I'm merging anyway
